.env.example

```
@@ -95,5 +99,5 @@ PHP_XDEBUG_REMOTE_HOST=host.docker.internal
 # running on the host.
 PHP_XDEBUG_REMOTE_PORT=9010
 # Logging is off by default as the log file may grow quite a bit in short time.
-#PHP_XDEBUG_REMOTE_LOG=/var/log/xdebug.log
+PHP_XDEBUG_REMOTE_LOG=/var/www/xdebug.log
```

I would suggest keeping the log disabled as per the comment.
```suggestion
#PHP_XDEBUG_REMOTE_LOG=/var/www/xdebug.log
```